store/sdk/src/test/java/org/apache/carbondata/sdk/file/ImageTest.java

```diff
@@ -815,4 +817,153 @@ public void binaryToCarbonWithHWD(String sourceImageFolder, String outputPath, S
     reader.close();
   }
 
+  @Test
+  public void testBinaryWithProjectionAndFileListsAndWithFile() throws Exception {
+    int num = 3;
```

Comment:
why this much big test case ?
have a folder with multiple carbondata files having one or two column. Read it with file, filelist, folder.

No need of image, decode logic and all.

Reply:
ajantha require merge to one test case.

remove write image and decode logic